packages/charts/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -898,17 +915,23 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
         legendColor: this.state.color,
         shouldHighlight,
       });
-      const barHeight: number = Math.max(yBarScale(point.y), 0);
-      let adjustedBarHeight = 0;
-      if (barHeight <= 0) {
-        return <React.Fragment key={point.x instanceof Date ? point.x.getTime() : point.x}> </React.Fragment>;
-      } else if (barHeight <= Math.ceil(yBarScale(this._yMax) / 100.0)) {
-        adjustedBarHeight = Math.ceil(yBarScale(this._yMax) / 100.0);
-      } else {
-        adjustedBarHeight = barHeight;
+      const barHeight: number = this.props.supportNegativeYValues
+        ? yBarScale(point.y) - yBarScale(0)
```

Comment:
why is yBarScale(0) needed?

Reply:
yBarScale(0) is not necessarlity 0. It depends on the domain and range we set. So to get relative height, we need it

Reply:
then you can't use 0 as input to get the baseline. If ymax < 0, then you will need to use yBarScale(yMax)

Reply:
Yes, correct. will add that condition.